frontend/src/scenes/annotations/AnnotationModal.tsx

```diff
@@ -109,24 +109,30 @@ export function AnnotationModal({
                                 {
                                     value: AnnotationScope.Insight,
                                     label: annotationScopeToName[AnnotationScope.Insight],
-                                    disabledReason:
-                                        !isInsightScoped && !onSavedInsight && 'You need to save the insight first.',
+                                    // if existing annotation data in db doesn't have insight id set on it
+                                    // we can't let them change scope to insight as we don't know which insight to map to
+                                    disabledReason: existingModalAnnotation
+                                        ? !hasInsightIdSetOnAnnotation &&
+                                          "Annotation wasn't originally scoped to an insight so we can't reduce scope to insight level"
```

Comment:
As we talked about: it would probably be more intuitive to re-scope to the insight being viewed